src/app/project/comment-period/review-comments-tab/review-comments-tab.component.ts

```diff
@@ -62,6 +62,11 @@ export class ReviewCommentsTabComponent implements OnInit, OnDestroy {
       name: 'Status',
       value: 'eaoStatus',
       width: 'col-2'
+    },
+    {
```

Comment:
This pushed the number of cols past 12 which I believe is the max for Bootstrap.

Reply:
Which one should we steal from?  Attachments?

Reply:
Not sure, they are a little different I believe. One of the other columns here just needs to be shrunk by one col point so that it doesn't cause an overflow when resizing. It looks like `dateAdded` is set to three, maybe that can come down to two?

Reply:
Fixed in 928612a